docs/reference/hook-command/list-of-hook-commands/credential-get.md

```diff
@@ -3,9 +3,6 @@
 ## Summary
 Access cloud credentials.
 
```

Comment:
Looks like a problem in the way we autogenerate this in Juju 4.

Reply:
Thanks for the initial assessment. @anvial @hpidcock @sinanawad please let us know if you think we can allocate any time to fix this.

Reply:
The answer to that is that Juju 3 uses https://github.com/juju/cmd/tree/v3 not the master branch, which contains fixes to always print the usage. I have a PR up [here](https://github.com/juju/cmd/pull/124) to forward merge to `master`. But that is largely useless in any case since Juju brought the cmd package into the `internal` folder, so I'd need to bring my fixes to cmd/v3 to the internal Juju cmd package.